lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -40,14 +40,18 @@ def create_vm(attrs)
 
             attrs[:comment] ||= ""
             attrs[:quota] = attrs[:quota].present? ? client.system_service.data_centers_service.data_center_service(datacenter).quotas_service.quota_service(attrs[:quota]).get : nil
+            cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => attrs[:sockets])
```

Comment:
You can move this into the if-statement below since it's not needed otherwise.

Reply:
DONE